pages/k8s/install-offline.md

```diff
@@ -212,7 +212,7 @@ from an internet connected machine:
 ```bash
 git clone https://github.com/charmed-kubernetes/cdk-shrinkwrap.git /tmp/.shrinkwrap
 cd /tmp/.shrinkwrap
-BUNDLE=cs:charmed-kubernetes-733       # Choose a deployment bundle (example is 1.21.x)
+BUNDLE=cs:charmed-kubernetes-862       # Choose a deployment bundle (example is 1.23.x)
```

Comment:
we want cs: here because we're explicitly talking about an old release; CH will use the CS shim to do the right thing when it encounters a cs:-prefixed artifact.